plugins/woocommerce/client/admin/client/marketplace/components/marketplace-link/marketplace-link.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import { Text } from '@woocommerce/experimental';
+import interpolateComponents from '@automattic/interpolate-components';
+import { Link } from '@woocommerce/components';
+import { getAdminLink } from '@woocommerce/settings';
+import { recordEvent } from '@woocommerce/tracks';
+
+interface MarketplaceLinkProps {
+	/**
+	 * HTML element to use for the Text component
+	 */
+	as?: string;
+	className?: string;
+	/**
+	 * The complete translatable string that includes {{sbLink}} and {{/sbLink}} placeholders
+	 * Example: "Visit the {{sbLink}}Official WooCommerce Marketplace{{/sbLink}} to find more tax solutions"
+	 */
+	translatedString: string;
+
+	eventName?: string;
+	marketplaceUrl: string;
+	/**
+	 * Optional callback function to be called when the link is clicked
+	 * If provided, this will be called instead of the default recordEvent behavior
+	 */
+	onClickCallback?: () => void;
+}
+
+/**
+ * A component that renders a link to the WooCommerce Marketplace with tracking.
+ */
+export const MarketplaceLink: React.FC< MarketplaceLinkProps > = ( {
+	as = '',
+	className = '',
+	translatedString,
+	eventName = '',
+	marketplaceUrl,
+	onClickCallback,
+} ) => (
+	<Text as={ as } className={ className }>
+		{ interpolateComponents( {
+			mixedString: translatedString,
+			components: {
+				sbLink: (
```

Comment:
What does sb stand for? :)

Reply:
Ha! I copied and modified an existing instance of this function, and it's to differentiate between two different links added in that case. When removing the one link I just didn't think to update the placeholder for the one that remained, but of course simply `Link` would do here.

Reply:
Agreed! Updating this to `Link` would make the uses later much clearer I think.

Reply:
Updated in 942d916